ext/odbc/odbc.stub.php

```diff
@@ -2,520 +2,442 @@
-function odbc_field_num($statement, string $field): int|false {}
-
-/** @param resource $odbc */
-function odbc_autocommit($odbc, ?bool $enable = null): int|bool {}
-
-/** @param resource $odbc */
-function odbc_commit($odbc): bool {}
-
-/** @param resource $odbc */
-function odbc_rollback($odbc): bool {}
-
-/** @param resource|null $odbc */
-function odbc_error($odbc = null): string {}
-
-/** @param resource|null $odbc */
-function odbc_errormsg($odbc = null): string {}
-
-/** @param resource $odbc */
-function odbc_setoption($odbc, int $which, int $option, int $value): bool {}
-
-/**
- * @param resource $odbc
- * @return resource|false
- */
-function odbc_tables($odbc, ?string $catalog = null, ?string $schema = null, ?string $table = null, ?string $types = null) {}
-
-/**
- * @param resource $odbc
- * @return resource|false
- */
-function odbc_columns($odbc, ?string $catalog = null, ?string $schema = null, ?string $table = null, ?string $column = null) {}
-
-/**
- * @param resource $odbc
- * @return resource|false
- */
-function odbc_gettypeinfo($odbc, int $data_type = 0) {}
-
-/**
- * @param resource $odbc
- * @return resource|false
- */
-function odbc_primarykeys($odbc, ?string $catalog, string $schema, string $table) {}
-
-#if !defined(HAVE_SOLID) && !defined(HAVE_SOLID_30) && !defined(HAVE_SOLID_35)
-/**
- * @param resource $odbc
- * @return resource|false
- */
-function odbc_procedurecolumns($odbc, ?string $catalog = null, ?string $schema = null, ?string $procedure = null, ?string $column = null) {}
-
-/**
- * @param resource $odbc
- * @return resource|false
- */
-function odbc_procedures($odbc, ?string $catalog = null, ?string $schema = null, ?string $procedure = null) {}
-
-/**
- * @param resource $odbc
- * @return resource|false
- */
-function odbc_foreignkeys($odbc, ?string $pk_catalog, string $pk_schema, string $pk_table, string $fk_catalog, string $fk_schema, string $fk_table) {}
-#endif
-
-/**
- * @param resource $odbc
- * @return resource|false
- */
-function odbc_specialcolumns($odbc, int $type, ?string $catalog, string $schema, string $table, int $scope, int $nullable) {}
-
-/**
- * @param resource $odbc
- * @return resource|false
- */
-function odbc_statistics($odbc, ?string $catalog, string $schema, string $table, int $unique, int $accuracy) {}
-
-#if !defined(HAVE_DBMAKER) && !defined(HAVE_SOLID) && !defined(HAVE_SOLID_30) &&!defined(HAVE_SOLID_35)
-/**
- * @param resource $odbc
- * @return resource|false
- */
-function odbc_tableprivileges($odbc, ?string $catalog, string $schema, string $table) {}
-
-/**
- * @param resource $odbc
- * @return resource|false
- */
-function odbc_columnprivileges($odbc, ?string $catalog, string $schema, string $table, string $column) {}
-#endif
-
-/* odbc_utils.c */
-
-function odbc_connection_string_is_quoted(string $str): bool {}
-
-function odbc_connection_string_should_quote(string $str): bool {}
-
-function odbc_connection_string_quote(string $str): string {}
+namespace ODBC {
```

Comment:
If follow the naming convention, wouldn't it be `Odbc`?

Reply:
Hm, yes, namespaces have different rules, but I am fine to rename it, especially because Tim's https://wiki.php.net/rfc/class-naming-acronyms will hopefully/likely pass.